charts/cloudzero-agent/values.yaml

```diff
@@ -48,9 +48,17 @@ prometheusConfig:
     additionalScrapeJobs: []
 
 kube-state-metrics:
-  enabled: false
+  enabled: true
+  fullnameOverride: "cloudzero-state-metrics"
   extraArgs:
     - --metric-labels-allowlist=pods=[app.kubernetes.io/component]
```

Comment:
Do we want this to be `*` instead of `app.kubernetes.io/component`

Reply:
I just left it as it was. What's the reasoning?

Reply:
I think we didn't want `*` initially for performance reasons, because the agent doesn't do well with potentially super high cardinatlity data

Reply:
I'm not sure I'm following - why do we need to change the documentation for configuring labels now that the CloudZero KSM is enabled by default?